apps/climatemappedafrica/src/components/HURUmap/IndicatorTitle/Action.js

```diff
@@ -79,15 +79,17 @@ function Action({ children, header, icon, title, id, ...props }) {
       >
         {({ TransitionProps }) => (
           <Fade {...TransitionProps} timeout={350}>
-            <ClickAwayListener onClickAway={handleClose}>
-              <Paper className={classes.paper}>
-                <ButtonBase onClick={handleClose} className={classes.header}>
-                  <Typography className={classes.title}>{header}</Typography>
-                  <CloseIcon />
-                </ButtonBase>
-                {children}
-              </Paper>
-            </ClickAwayListener>
+            <div>
```

Comment:
What does this extra `div` solve?

Reply:
Then include a comment with a link to the issue @kelvinkipruto ... otherwise, the code doesn't make sense.